### PR TITLE
Addition of Fortaleza/Brazil orthophotos

### PR DESCRIPTION
**Pull request by matheusgomesms**
Addition of Orthophotos explicitly available for OSM usage

### CODE REVIEW

**Comment by matheusgomesms** [comment]
Hi there,

The OSM community received explicit permission to use all data from Spatial Data Infrastructure - Municipality of Fortaleza/Brazil (transcription and English translation available [here](https://wiki.openstreetmap.org/wiki/File:Autoriza%C3%A7%C3%A3o_IDE_SEFIN_Fortaleza.png)).

In that SDI we have access to a couple of data, including orthophotos at 10 cm resolution, so I believe it would be great to have them on OSM (already available on JOSM).

Please let me know if there are any errors (first PR here), or if you need more info.

Best,
Matheus

**Comment by matheusgomesms** [comment]
I fixed and updated the file. Please use most recent commit (4d828e2).

Best,
Matheus

File: sources/south-america/br/Fortaleza_2016_Orthophoto.geojson [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "type" : "Feature",
+  "properties" : {
+    "id" : "Fortaleza_Ortofoto_2016",
+    "url" : "https://geoserver.sefin.fortaleza.ce.gov.br/geoserver/IMAGEAMENTO/wms?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=ortofoto_2016&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
+    "type" : "wms",
+    "country_code": "BR",
+    "name" : "Fortaleza Ortofoto 2016",
+    "permission_osm": "explicit",
+    "license_url": "https://wiki.openstreetmap.org/wiki/File:Autoriza%C3%A7%C3%A3o_IDE_SEFIN_Fortaleza.png",
+    "best": "true",
```
Comment (rbuffat) [inline sources/south-america/br/Fortaleza_2016_Orthophoto.geojson]: This should be `"best": true` to pass the tests.

File: sources/south-america/br/Fortaleza_2016_Orthophoto.geojson [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "attribution": {
+      "url": "https://ide.sefin.fortaleza.ce.gov.br/",
+      "text": "Prefeitura Municipal de Fortaleza"
+    },
+    "available_projections" : [
+            "EPSG:31984",
+            "EPSG:4326"
+    ],
+    "category": "photo",
+    "valid-georeference": "true"
```
Comment (rbuffat) [inline sources/south-america/br/Fortaleza_2016_Orthophoto.geojson]: Same here: "valid-georeference": true

**Review by rbuffat — COMMENTED** [review]
Thanks for creating this PR! I made some comments for small changes so that the JSON corresponds to the correct format.

**Comment by matheusgomesms** [comment]
Hi René, 

Thanks a lot for taking a look into the small problems! I fixed them, so it should be okay now.

Best,
Matheus

**Comment by matheusgomesms** [comment]
Added Privacy Policy URL. Now it should pass strict check

**Pull request merged**